packages/react-divider/src/components/Divider/renderDivider.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { getSlots } from '@fluentui/react-utilities';
+import { DividerState } from './Divider.types';
+import { dividerShorthandProps } from './useDivider';
+
+/**
+ * Function that renders the final JSX of the component
+ */
+export const renderDivider = (state: DividerState) => {
+  const { slots, slotProps } = getSlots(state, dividerShorthandProps);
+  const { children } = state;
+
+  const contentVisible = children || slotProps.content?.children;
+
+  return contentVisible ? (
+    <slots.root {...slotProps.root} role="separator">
+      <div>{children}</div>
+    </slots.root>
+  ) : (
+    <slots.root {...slotProps.root} role="separator" />
+  );
```

Comment:
Would this be simpler ? 

```suggestion
  return  (
    <slots.root {...slotProps.root} role="separator">
      {contentVisible &&<div>{children}</div>}
    </slots.root>
  ) 
```

Reply:
Updated